data/data.js

```diff
@@ -1,14 +1,19 @@
+import path from 'path';
+import {fileURLToPath} from 'url';
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```

Comment:
@GGCristo: The `process.cwd()` didn't work when in an arbitrary path so I had to use `__dirname` fron `path`